scripts/build.py

```diff
@@ -0,0 +1,39 @@
+from utils import (
+    get_manim_plugins,
+    parse_examples_folder,
+    write_plugins_to_json,
+    write_governance_file,
+    render_manim_example,
+    write_example_json,
+    guarantee_existence_of_folders,
+)
+from cli import parser
+
+config = parser.parse_args()
```

Comment:
Do we really need a separate module for parsing a single CLI function? Instead, why not just move that part of the code into this file and remove that module?

Reply:
Done. Removed this module and shifted it to build.py